modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/home/UserSubscriptions.es.js

```diff
@@ -60,6 +60,12 @@ export default withRouter(({history, location}) => {
 		}
 	);
 
+	if (threads && threads.myUserAccountSubscriptions.items) {
+		threads.myUserAccountSubscriptions.items = threads.myUserAccountSubscriptions.items.filter(
+			(thread) => thread.graphQLNode.showAsQuestion
+		);
+	}
+
```

Comment:
I'm worry about the fact that this may cause problem with Pagination, since this code is filtering only the items from the API.

I'll confirm the message boards api's to see if it's possible to filter showAsQuestions throw the API Request